### PR TITLE
Refactor/register

### PR DESCRIPTION
**Pull request by Saksham-1508**

### CODE REVIEW

**Review by divyanshkhetan — CHANGES_REQUESTED** [review]
Also please resolve your merge conflicts apart from the requested changes. Thanks!

File: templates/AlumniConnect/registration_form.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+    <div id="id_checkbox_update" class="form-check"> 
+        <label for="checkbox_update" class="form-check-label requiredField">
+            <input type="checkbox" name="checkbox_update" class="checkboxinput form-check-input" required=""
+                id="checkbox_update"/>
+            I will update my information at regular intervals and will engage in the Alumni network
+            actively.<span class="asteriskField">*</span> 
+        </label> 
+    </div>
+</div>
+<button type="submit" class="btn btn-primary">Submit</button>
```
[the remark this answers is not in the document]
Comment (Saksham-1508) [reply templates/AlumniConnect/registration_form.html:311]: I've added the changes to make the form responsive

File: templates/AlumniConnect/registration_form.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    <div class="custom-file">
+        <label class="custom-file-label" for="profile-picture">Choose Profile Picture</label>
+        <input type="file" class="custom-file-input" name="profile_picture" accept="image/*" required=""
+            id="profile-picture" value="{{profile_picture}}"/>
+    </div>
+</div>
+<div class="form-group">
+    <div id="id_checkbox_update" class="form-check"> 
+        <label for="checkbox_update" class="form-check-label requiredField">
+            <input type="checkbox" name="checkbox_update" class="checkboxinput form-check-input" required=""
```
[the remark this answers is not in the document]
Comment (Saksham-1508) [reply templates/AlumniConnect/registration_form.html]: Done

**Comment by garg3133** [comment]
Added this as part of #122.

Thanks, @Saksham-1508 for this new page and @divyanshkhetan for reviews.

**Pull request closed without merging**